config/mainCfg_ETau_UL17.cfg

```diff
@@ -0,0 +1,67 @@
+[general]
+lumi = 41529 # pb^-1 full 2017
+
+data = EGamma_Run2017B, EGamma_Run2017C, EGamma_Run2017D, EGamma_Run2017E, EGamma_Run2017F
+
+signals = GGF_Radion250, GGF_Radion260, GGF_Radion270, GGF_Radion280, GGF_Radion300, GGF_Radion320, GGF_Radion350, GGF_Radion400, GGF_Radion450, GGF_Radion500, GGF_Radion550, GGF_Radion600, GGF_Radion650, GGF_Radion700, GGF_Radion750, GGF_Radion800, GGF_Radion850, GGF_Radion900, GGF_Radion1000, GGF_Radion1250, GGF_Radion1500, GGF_Radion1750, GGF_Radion2000, GGF_Radion2500, GGF_Radion3000
+
+backgrounds = DY_NLO_incl_stitch, DY_LM, DY_NLO_0J, DY_NLO_1J, DY_NLO_2J, DY_NLO_Pt0To50, DY_NLO_Pt50To100, DY_NLO_Pt100To250, DY_NLO_Pt250To400, DY_NLO_Pt400To650, DY_NLO_Pt650ToInf, TTfullyHad, TTfullyLep, TTsemiLep, WJets_HT_0_70, WJets_HT_70_100, WJets_HT_100_200, WJets_HT_200_400, WJets_HT_400_600, WJets_HT_600_800, WJets_HT_800_1200, WJets_HT_1200_2500, WJets_HT_2500_Inf, EWKWMinus2Jets_WToLNu, EWKWPlus2Jets_WToLNu, EWKZ2Jets_ZToLL, TWtop, TWantitop, singleTop_top, singleTop_antitop, ZH_HTauTau, ZH_HBB_ZLL, ZH_HBB_ZQQ, WplusHTauTau, WminusHTauTau, TTWW, TTWZ, TTZZ, TTWJetsToQQ, ggHTauTau, VBFHTauTau, WWW_1, WWW_2, WWZ_1, WWZ_2, WZZ_1, WZZ_2, ZZZ_1, ZZZ_2, WW, WZ, ZZ, ttHJetTononBB, ttHJetToBB, TTWJetsToLNu, TTZToQQ, TTZToLLNuNu, ttHJetToTauTau
+
+variables = dau1_pt, dau2_pt, dau1_eta, dau2_eta, dau1_phi, dau2_phi, ditau_deltaR, bjet1_pt, bjet1_bID_deepFlavor, bjet2_pt, bjet2_bID_deepFlavor, bjet1_eta, bjet2_eta, tauH_mass, tauH_pt, tauH_SVFIT_mass, tauH_SVFIT_pt, bH_mass, bH_pt, HH_mass, HH_pt, HHKin_mass, dau1_iso, dau2_iso, dib_deltaR, btau_deltaRmin, btau_deltaRmax, HH_deltaR, dau1_deepTauVsMu, dau1_deepTauVsEle, dau1_deepTauVsJet, dau2_deepTauVsMu, dau2_deepTauVsEle, dau2_deepTauVsJet, HT20Full, DNNoutSM_kl_1, metnomu_et, metnomu_phi, met_phi, met_et, njets, njets20, njets50, bjet1_JER, bjet2_JER
+
+selections = baseline, baseline_boosted, res1b, res2b, boosted
+
+regions    = SR, SStight, OSinviso, SSinviso
+
+[configs]
+sampleCfg = config/sampleCfg_UL17.cfg
+cutCfg    = config/selectionCfg_ETau_UL17.cfg
+#pattern  = goodsystfiles  # use this only when running on systematics files
+
+[merge_plots]
+data_obs    = EGamma_Run2017B, EGamma_Run2017C, EGamma_Run2017D, EGamma_Run2017E, EGamma_Run2017F
```

Comment:
For the plots I would propose we limit the list to data_obs, TT,W,DY,singleH, other.